mmv1/products/alloydb/User.yaml

```
@@ -0,0 +1,111 @@
+import_format: ['projects/{{project}}/locations/{{location}}/clusters/{{cluster}}/users/{{user_id}}']
+skip_sweeper: true
+autogen_async: true
+custom_code: !ruby/object:Provider::Terraform::CustomCode
+  custom_import: templates/terraform/custom_import/alloydb_user.go.erb
+examples:
+  - !ruby/object:Provider::Terraform::Examples
+    name: 'alloydb_user_builtin'
+    primary_resource_id: 'user1'
+    vars:
+      alloydb_cluster_name: 'alloydb-cluster'
+      alloydb_cluster_pass: 'cluster_secret'
+      alloydb_cluster_resource_id: 'default'
+      alloydb_instance_name: 'alloydb-instance'
+      alloydb_instance_resource_id: 'default'
+      alloydb_user_name: 'user1'
+      alloydb_user_pass: 'user_secret'
+      network_name: 'alloydb-network'
+    test_vars_overrides:
+      alloydb_cluster_name: '"tf-test-user-cluster-" + acctest.RandString(t, 10)'
+      alloydb_cluster_pass: '"cluster_secret1"'
+      alloydb_cluster_resource_id: '"test-cluster1"'
+      alloydb_instance_name: '"tf-test-instance1"'
+      alloydb_instance_resource_id: '"test-instance1"'
+      alloydb_user_name: '"user_1"'
+      alloydb_user_pass: '"user_1_pass"'
+      network_name: '"alloydb-user-builtin-" + acctest.RandString(t, 10)'
+    ignore_read_extra:
+      - 'password'
+  - !ruby/object:Provider::Terraform::Examples
+    name: 'alloydb_user_iam'
+    primary_resource_id: 'user2'
+    vars:
+      alloydb_cluster_name: 'alloydb-cluster'
+      alloydb_cluster_resource_id: 'default'
+      alloydb_instance_name: 'alloydb-instance'
+      alloydb_instance_resource_id: 'default'
+      alloydb_cluster_pass: 'cluster_secret'
+      alloydb_user_name: 'user2@foo.com'
+      network_name: 'alloydb-network'
+    test_vars_overrides:
+      alloydb_cluster_name: '"tf-test-user-cluster-" + acctest.RandString(t, 10)'
+      alloydb_cluster_pass: '"cluster_secret2"'
+      alloydb_cluster_resource_id: '"test-cluster2"'
+      alloydb_instance_name: '"tf-test-instance2"'
+      alloydb_instance_resource_id: '"test-instance2"'
+      alloydb_user_name: '"user2@foo.com"'
+      network_name: '"alloydb-user-iam-" + acctest.RandString(t, 10)'
+parameters:
+  - !ruby/object:Api::Type::ResourceRef
+    name: 'cluster'
+    description: |
+      Identifies the alloydb cluster. Must be in the format
+      'projects/{project}/locations/{location}/clusters/{cluster_id}'
+    required: true
+    immutable: true
+    resource: 'Cluster'
+    imports: 'name'
+    url_param_only: true
+  - !ruby/object:Api::Type::String
+    name: 'userId'
+    required: true
+    immutable: true
+    url_param_only: true
+    description: |
+      The database role name of the user.
+  - !ruby/object:Api::Type::Enum
+    name: 'userType'
+    required: true
+    immutable: true
+    description: |
+      The type of this user.
+    values:
+      - :ALLOYDB_BUILT_IN
+      - :ALLOYDB_IAM_USER
+properties:
+  - !ruby/object:Api::Type::String
+    name: 'password'
```

```suggestion
properties:
  - !ruby/object:Api::Type::String
    name: 'name'
    output: true
    description: |
      Name of the resource in the form of projects/{project}/locations/{location}/cluster/{cluster}/users/{user}.
  - !ruby/object:Api::Type::String
    name: 'password'
```
Let's still add this field. I think we may still need the name field at some point